vignettes/dark_ink_transition/lib/transition_container.dart

```diff
@@ -7,31 +7,31 @@ import 'main.dart';
 
 class TransitionContainer extends StatefulWidget {
 
-  final ValueNotifier<bool> darkModeValue;
-  final Widget child;
+  final ValueNotifier<bool>? darkModeValue;
+  final Widget? child;
 
   TransitionContainer({ this.darkModeValue, this.child });
 
   @override
   State createState() {
-    return _TransitionContainerState(darkModeValue, child);
+    return _TransitionContainerState(darkModeValue!, child!);
```

Comment:
Should just be marking the `darkModeValue` as `required` here, then no need for `!` or `?` operators.

Same with `child`.